### DISCUSSION TITLE
Question about Table View Rows!

### DISCUSSION THREAD

**Discussion by chlghdwns** [slint-ui/slint#5067] [Q&A]
![image](https://github.com/slint-ui/slint/assets/111585117/7169e6f6-afb1-4ffa-bb66-e86fae5ddc2b)

First of all, I'm Korean, so I'm using English through a translator, so please understand that I'm not good at it.

Do you know how to put row data in the table shown in the figure with data generated through iterations such as the 'for' statement?

**Comment by hunger** [comment]
This is the code generating that TableView:

https://github.com/slint-ui/slint/blob/c399447c932dc74be3165a8f670285d4671e5845/examples/gallery/main.rs#L26-L38

Don't be worry about English! I am German. I can not speak English properly either. You will fit right in:-)

**Reply by chlghdwns** [reply]
Thank you sir ^_^

**Comment by syf20020816** [comment]
if you wanna a flexible table, you can try this : https://surrealism-all.github.io/SurrealismUI.github.io/components/data/table.html
